vk_video_decoder/include/vkvideo_parser/VulkanVideoParserIf.h

```diff
@@ -626,7 +494,7 @@ class VkParserVideoDecodeClient {
         = 0; // Returns max number of reference frames
         // (always at least 2 for MPEG-2)
     virtual bool
-    AllocPictureBuffer(VkPicIf** ppPicBuf)
+    AllocPictureBuffer(VkPicIf** ppPicBuf, uint32_t codedWidth, uint32_t codedHeight)
```

Comment:
I see why we need these fields, but I don't think this information is needed. Images are allocated with the maximum size required, and they will not get allocated on this request. I don't think we need to change this prototype.